src/Symfony/Component/Workflow/Workflow.php

```diff
@@ -61,6 +61,9 @@ public function getMarking($subject)
                 throw new LogicException(sprintf('The Marking is empty and there is no initial place for workflow "%s".', $this->name));
             }
             $marking->mark($this->definition->getInitialPlace());
+
+            // Because the marking could have been initialized, we update the subject
```

Comment:
I would remove this comment because now, it's clear the marking has been updated.